### PR TITLE
[fix](cloud cluster) Fix if clusterName null select throw npe

### PR DESCRIPTION
**Pull request by deardeng**
Fix clusterName null, select throw npe
```
java.lang.NullPointerException: Cannot invoke "Object.hashCode()" because "key" is null
        at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936) ~[?:?]
        at java.util.concurrent.ConcurrentHashMap.getOrDefault(ConcurrentHashMap.java:1594) ~[?:?]
        at org.apache.doris.system.SystemInfoService.getCloudStatusByName(SystemInfoService.java:258) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectContext.cloudNoBackendsReason(ConnectContext.java:963) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.planner.OlapScanNode.addScanRangeLocations(OlapScanNode.java:853) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.planner.OlapScanNode.computeTabletInfo(OlapScanNode.java:1136) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.planner.OlapScanNode.createScanRangeLocations(OlapScanNode.java:965) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.planner.OlapScanNode.finalize(OlapScanNode.java:612) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.planner.OriginalPlanner.createPlanFragments(OriginalPlanner.java:210) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.planner.OriginalPlanner.plan(OriginalPlanner.java:101) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.analyzeAndGenerateQueryPlan(StmtExecutor.java:1399) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.analyze(StmtExecutor.java:1196) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.executeByLegacy(StmtExecutor.java:859) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:569) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:509) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:492) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.dispatch(ConnectProcessor.java:640) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.processOnce(ConnectProcessor.java:909) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### CODE REVIEW

**Comment by deardeng** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by deardeng** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java [hunk 1]
```diff
@@ -1113,9 +1113,13 @@ public String toString() {
 
     public static String cloudNoBackendsReason() {
         StringBuilder sb = new StringBuilder();
-        sb.append(" ");
         if (ConnectContext.get() != null) {
             String clusterName = ConnectContext.get().getCloudCluster();
+            String hits = "or you may not have permission to access the current cluster = ";
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java:1118]: typo "hint"?

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Comment by dataroaring** [comment]
run performance

**Pull request merged**